src/contact.ts

```diff
@@ -477,6 +477,37 @@ export class Contact implements Sayable {
    */
   public alias(empty: null): Promise<boolean>
 
+  /**
+   * GET / SET / DELETE the alias for a contact
+   *
+   * @param {(none | string | null)} newAlias ,
+   * @returns {(string | null | Promise<boolean>)}
+   *
+   * @example GET the alias for a contact
+   * ```ts
+   * const alias = contact.alias()  //@returns {(string | null)}
+   * ```
+   *
+   * @example SET the alias for a contact
+   * ```ts
+   * const ret = await contact.alias('lijiarui')
+   * if (ret) {
+   *   console.log(`change ${contact.name()}'s alias successfully!`)
+   * } else {
+   *   console.error('failed to change ${contact.name()}'s alias!')
```

Comment:
You should change **'** to **`** here, or it may cannot be parsed successfully.
```ts
console.error(`failed to change ${contact.name()}'s alias!`)
```